release-notes/EPM/I-DOCSIS/EPM_7.0.1_I-DOCSIS.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: EPM_7.0.1_I-DOCSIS
+---
+
+# EPM 7.0.1 I-DOCSIS - preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be modified or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### Channel Utilization added for US/DS Service Group [ID_38849]
+
+The ​Upstream and Downstream Service Group entities now display the Channel Utilization average of the associated channels in the data section.
+
+## Changes
+
+#### EPM_I_DOCSIS_SetThresholdsTableToCollectors script ID updated [ID_39040]
+
+The ID of the *EPM_I_DOCSIS_SetThresholdsTableToCollectors* script has been updated to prevent possible conflicts with other script IDs.
```

Comment:
@sergiosuarezsky, would users have experienced the conflict with other script IDs as a bug? If yes, what would this bug look like for them?

Reply:
Hello @MariekeGO, I would say yes, is likely that other script ID po up that error. When looking at the automation log, the user will see this: "Execute ID 1 is used by another script"

Reply:
Since this means users will probably look for this in the "Fixes" section, I'll rephrase this one as a bug fix.